bin/oneoff/import_unit_details.rb

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'optparse'
+require_relative '../../deployment'
+
+# Once this script is ready, only levelbuilder should be added to this list.
+raise unless [:development, :adhoc].include? rack_env
+
+$verbose = false
+def log(str)
+  puts str if $verbose
+end
+
+# Wait until after initial error checking before loading the rails environment.
+def require_rails_env
+  log "loading rails environment..."
+  start_time = Time.now
+  require_relative '../../dashboard/config/environment'
+  log "rails environment loaded in #{(Time.now - start_time).to_i} seconds."
+end
+
+def parse_options
+  OpenStruct.new.tap do |options|
+    options.local = false
+    options.unit_name = nil
+    options.dry_run = false
+
+    opt_parser = OptionParser.new do |opts|
+      opts.banner = <<~BANNER
+
+        Usage: #{$0} [options]
+
+        Example: runner.rb -u coursea-2021
+        Example: runner.rb -l -u csd1-2021
+        Example: runner.rb -l -u csp2-2021,csp3-2021,csp4-2021
+      BANNER
+
+      opts.separator ""
+
+      opts.on('-l', '--local', 'Use local curriculum builder running on localhost:8000.') do
+        options.local = true
+      end
+
+      opts.on('-u', '--unit_names UnitName1,UnitName2', Array, 'Unit names to import') do |unit_names|
+        options.unit_names = unit_names
+      end
+
+      opts.on('-n', '--dry-run', 'Perform basic validation without importing any data.') do
+        options.dry_run = true
+      end
+
+      opts.on('-v', '--verbose', 'Use verbose debug logging.') do
+        $verbose = true
+      end
+
+      opts.on_tail("-h", "--help", "Show this message") do
+        puts opts
+        exit
+      end
+    end
+
+    opt_parser.parse!(ARGV)
+  end
+end
```

Comment:
Options parsing is mostly copied from here: https://github.com/code-dot-org/code-dot-org/blob/3c676c26a7f2c897f025bc8fcff167d80c97313e/dashboard/test/ui/runner.rb#L91